documentation/modules/proc-jmxtrans-deployment.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='proc-jmxtrans-deployment-{context}']
+= JmxTrans deployment configuration.
+
+JmxTrans deployment can be configured using the `jmxTrans` property in the following resources:
+
+* `Kafka.spec.kafka.jmxOptions`
+
+The JmxTrans deployment can read from a secure or insecure Kafka broker.
+
+To configure JmxTrans deployment, the following properties must be defined:
+
+* `Kafka.spec.kafka.jmxOptions.jmxTrans.outputDefinitions`
+* `Kafka.spec.kafka.jmxOptions.jmxTrans.queries`
+
+
+.Configuring JmxTrans output definitions
+
+Output definitions specify where the JMX metrics get pushed to and the data format it is pushed in.
+For information about supported data formats see link:https://github.com/jmxtrans/jmxtrans/wiki/OutputWriters[Data formats].
+How many seconds JmxTrans agent will wait for before pushing new data can be configured through the `flushDelay` property.
+The `host` and `port` property defines the target host address and target port the data is pushed to.
+The `name` property is a required property which is referenced by `Kafka.spec.kafka.jmxOptions.jmxTrans.queries` property.
+
+An example configuration pushing JMX data in the Graphite format every 5 seconds to a Logstash database on "http://myLogstash:9999" and another pushing to standardOutput:
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    jmxOptions:
+      # ...
+      jmxTrans:
+        outputDefinitions:
+          - outputType: "com.googlecode.jmxtrans.model.output.GraphiteWriter"
+            host: "http://myLogstash"
+            port: 9999
+            flushDelay: 5
+            name: "logstash"
+          - outputType: "com.googlecode.jmxtrans.model.output.StdOutWriter"
+            name: "standardOut"
+        # ...
+    # ...
+  zookeeper:
+    # ...
+----
+
+.Configuring JmxTrans queries
+JmxTrans queries specify what JMX metrics are read from the Kafka brokers.
+Configure the `targetMBean` field to specify which target MBean on the Kafka broker is addressed.
+Configuring the `attributes` field specify which MBean attribute is JMX metrics read from the target MBean.
```

Comment:
_Suggested change:_

Configuring the `attributes` field specify which MBean attribute is JMX metrics read from the target MBean.

`>>`

Configuring the `attributes` field specifies which MBean attribute is read from the target MBean.